src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -220,9 +220,10 @@ function getDataWithKeys(data, query, options = {}) {
       return row;
     }
 
-    const key = fields.length
-      ? fields.map(field => getLabel(row[field], options)).join(',')
-      : aggregate;
+    const key =
+      fields && fields.length
```

Comment:
Can revert this change

Reply:
Yeah, it's not a necessary check, it'll never be falsy otherwise there's big problems